.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@xtfer, you forgot to add .idea to your global .gitignore file. :smile:  Please rebase this out.

Reply:
Whoops. Probably the result of changing machines.